config/locales/en.yml

```diff
@@ -253,6 +254,7 @@ en:
         lead_time:
           label: "Lead time in days"
           custom_error: "Enter the approximate lead time in days, do not include words or symbols. If the product is available now, enter 0"
+          suffix: "Days"
```

Comment:
```suggestion
          suffix: "days"
```

Reply:
Thanks Bruce, i've changed them both to be lowercase